spv/sync.go

```diff
@@ -963,7 +1057,7 @@ func (s *Syncer) getHeaders(ctx context.Context, rp *p2p.RemotePeer) error {
 			s.locatorMu.Unlock()
 			continue
 		}
-
+		s.fetchHeadersProgress(int32(added), headers[len(headers)-1].Timestamp.UnixNano())
```

Comment:
UnixNano returns the nanoseconds of the time.Time.  You want Unix instead, which returns seconds since the epoch.